dclab/rtdc_dataset/fmt_hdf5.py

```diff
@@ -198,6 +198,9 @@ def __init__(self, h5path, *args, **kwargs):
 
         # Parse configuration
         self.config = RTDC_HDF5.parse_config(h5path)
+        # Add "user" section if it doesn't exist
```

Comment:
Adding the user section should not be necessary here. This must be handled globally in config.py `Configuration.__getitem__` (Otherwise user sections won't work with e.g. DCOR or dictionary data).